llap-tez/src/java/org/apache/hadoop/hive/llap/tezplugins/metrics/LlapTaskSchedulerMetrics.java

```diff
@@ -254,6 +272,14 @@ public void setWmUnusedGuaranteed(int unusedGuaranteed) {
     wmUnusedGuaranteedCount.set(unusedGuaranteed);
   }
 
+  public void addTaskLatency(String key, long value) {
```

Comment:
Rename key to daemonID and/or add comment, describing the parameters.

Reply:
Done